apollo-router-core/src/query_planner/snapshots/apollo_router_core__query_planner__bridge_query_planner__tests__plan_root.snap

```diff
@@ -10,5 +11,6 @@ Fetch(
         operation: "{me{name{first last}}}",
         operation_name: None,
         operation_kind: Query,
+        id: None,
```

Comment:
What is the id ? What does it represent ?

Reply:
that's a leftover from the query plan changes I did in https://github.com/apollographql/router/pull/1175/commits/5d2a859027111363861fa973c2665a5d413c965f but since removed (they will come in a followup PR). I don't know why it remained in the snapshots though

Reply:
fixed in a6aa2fbe